rebaser.go

```diff
@@ -24,6 +24,20 @@ func (r *Rebaser) Rebase(
 		return errors.Wrap(err, "get image metadata")
 	}
 
+	workingImageStack, err := metadata.GetStackMetadata(workingImage)
+	if err != nil {
+		return errors.Wrap(err, "get working image stack")
+	}
+
+	newBaseImageStack, err := metadata.GetStackMetadata(newBaseImage)
+	if err != nil {
+		return errors.Wrap(err, "get new base image stack")
+	}
+
+	if workingImageStack.Id != newBaseImageStack.Id {
+		return errors.New("incompatible stack")
```

Comment:
We should probably output a little bit more information. Something like `incompatible stack: stack 'stack.new' is not compatible with 'stack.original'`

Reply:
Done